civicrm/custom/ext/gov.nysenate.webintegration/info.xml

```diff
@@ -11,14 +11,17 @@
   <urls>
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate>2017-02-20</releaseDate>
-  <version>1.0</version>
+  <releaseDate>2021-12-06</releaseDate>
+  <version>1.2</version>
```

Comment:
this doesn't align with the version declared in the upgrade script. I think you want it 1.1